si2.dal/Entities/Address.cs

```diff
@@ -32,11 +31,11 @@ public class Address : Si2BaseDataEntity<Guid>, IAuditable
         public string CountryAr { get; set; }
 
         [Required]
-        [Column(TypeName = "decimal(9,6)")]
+        [Column(TypeName = "decimal(10,7)")]
```

Comment:
Shouldn't we add these lines here too :
[RegularExpression(@"^-?\d+\.\d{0,7}$", ErrorMessage = "Please enter up to 9 decimal digits for Longitude")]
[Range(-90, 90)]
like we did in the dtos?